nf_core/components/update.py

```diff
@@ -879,25 +879,23 @@ def update_linked_components(self, modules_to_update, subworkflows_to_update, up
             if m_update in updated:
                 continue
             original_component_type, original_update_all = self._change_component_type("modules")
-            self.update(m_update, silent=True, updated=updated, check_diff_exist=check_diff_exist)
-            self._reset_component_type(original_component_type, original_update_all)
+            try:
+                self.update(m_update, silent=True, updated=updated, check_diff_exist=check_diff_exist)
+            except LookupError as e:
+                # If the module to be updated is not available, check if there has been a name change
+                if "not found in list of available" in str(e):
+                    # Skip update, we check for name changes with manage_changes_in_linked_components in line #261
```

Comment:
```suggestion
                    # Skip update, we check for name changes with manage_changes_in_linked_components
```
I am afraid that line number reference will be outdated soon, so just referencing the method should be good enough

Reply:
I will change it in #2177 and merge that one as you reviewed here already 😄